src/components/routes/Host/ListingForm/ListingHelpText.tsx

```diff
@@ -0,0 +1,232 @@
+  </div>
+);
+
+const SecurityDepositUsd = () => (
+  <div>
+    <h3>Security Deposit</h3>
+    <p><span>Note:</span> Security deposit is not charged at the time of booking. It will only be charged if a guest damages your property.</p>
+  </div>
+);
+
+const IcalUrls = () => (
+  <div>
+    <h3>iCal Urls</h3>
+    <p>To avoid double bookings and to prevent multiple guests from booking the same dates, sync your Beenest calendar with your other calendars and input your iCal URL from other booking platforms.</p>
+    <p>Instructions to export calendars on other platforms are generally:</p>
+    <ol>
+      <li>Go to <span>Host</span> and select <span>Calendar</span></li>
+      <li>Click <span>Availability</span> settings in your calendar view</li>
+      <li>Under Sync calendars, select <span>Export Calendar</span></li>
+      <li>Copy and paste the calendar link into Beenest</li>
+    </ol>
+    <p><span>Note:</span> When you edit an external calendar that syncs with your calendar on Beenest, it will take a few hours for those changes to be visible to guests who view your listing.</p>
+  </div>
+);
+
+const CheckInTime = () => (
+  <div>
+    <h3>Check-in (From) and (To)</h3>
+    <p>Provide a window of time in which guests are able to check into your place. Typical times guests check in are between 11 AM - 2PM.</p>
+    <p>Allowing guests to check in later, for instance up until 11:30PM, is preferred.</p>
+  </div>
+);
+
+const CheckOutTime = () => (
+  <div>
+    <h3>Check-out (Before)</h3>
+    <p>Inform guests the time in which they are required to exit the premises. Typical Check-out time is 3 PM.</p>
+  </div>
+);
+
+const HouseRules = () => (
+  <div>
+    <h3>House Rules</h3>
+    <h4>Keep your guests as informed as possible and set ground rules.</h4>
+    <p>Common house rules include:</p>
+    <ul>
+      <li>No pets</li>
+      <li>No smoking</li>
+      <li>No parties or events</li>
+      <li>Self check-in with keypad</li>
+      <li>Additional cancellation policies</li>
+    </ul>
+  </div>
+);
+
+const ListingHelpTextInterface: ListingHelpTextInterfaceInterface = {
+  homeType: <HomeType />,
+  title: <Title />,
+  description: <Description />,
+  addressLine1: <FullAddress/>,
+  addressLine2: <FullAddress/>,
+  city: <FullAddress/>,
+  state: <FullAddress/>,
+  postalCode: <FullAddress/>,
+  country: <FullAddress />,
+  listingPicUrl: <ListingPicUrl />,
+  photos: <Photos />,
+  
+  sleepingArrangement: <SleepingArrangement />,
+  numberOfBedrooms: <NumberOfBedrooms />,
+  numberOfBathrooms: <NumberOfBathrooms />,
+  sharedBathroom: <SharedBathroom />,
+  amenities: <Amenities />,
+
+  maxGuests: <MaxGuests />,
+  minimumNights: <MinimumNights />,
+  pricePerNightUsd: <PricePerNightUsd />,
+  securityDepositUsd: <SecurityDepositUsd />,
+  icalUrls: <IcalUrls />,
+  
+  checkInTime: <CheckInTime />,
+  checkOutTime: <CheckOutTime />,
+  houseRules: <HouseRules />,
```

Comment:
I really like this approach!

Would `[ListingField.HOUSE_RULES]: <HouseRules />` make sense here?

Reply:
👍  Oh ya definitely could use that here as well